.run/usermgt(development).run.xml

```diff
@@ -0,0 +1,67 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I think thats something we dont want to check in?

Reply:
I removed it.